### ISSUE TITLE
CLIMSOFT User Manual

### ISSUE THREAD

**Issue by rdstern** [opencdms/Climsoft#327]
This is a really important document.  It is perfect for the testing process.  Indeed I am not sure how we (or others could have proceeded without it.  Congratulations to Marcellin and Samuel for producing it.

I went through the guide, before the testing starts in Western Kenya.  It is good.  I had - of course - a few questions, that I have inserted in the document.  I also have made a number of small edits to the wording, and these are in tracked changes.  So you can go through and accept the ones you are happy with, and reject the others.
[ClimsoftV4_User_Manual_updated22082018RDS.docx](https://github.com/opencdms/Climsoft/files/2321469/ClimsoftV4_User_Manual_updated22082018RDS.docx)

If you are able to critique my changes quickly, that would be ideal.  Otherwise you may wish to wait until the end of the week, when I can send another copy with more suggestions.

**Comment by maxwellfundi** [comment]
I agree this is a very important document. We are now having a go at it so that we can get climsoft set up ready for testing.

**Comment by rdstern** [comment]
In addition to the small edits noted above it will be good to consider larger changes.
I give just one example, because it is attractive and prominent in the CLIDATA manual.  They show the following picture of a station:
![image](https://user-images.githubusercontent.com/11226469/44945048-33996700-add9-11e8-8db3-683938feb473.png)

I suggest this looks good - and this is the competition for CLIMSOFT.
I was about to make an issue that this would be a great feature to add into CLIMSOFT.  Then I noticed, in Section 5.2.7, you already have this:
![image](https://user-images.githubusercontent.com/11226469/44945088-912db380-add9-11e8-9e31-060398fd207b.png)

Now compare the two pictures and text.  In the CLIMSOFT manual there is no more text about this section.  So I really don't know how to complete this dialogue.  What is Feature Class (where you have entered 2)?  And what does 2 mean?  And then there is "building", but the picture doesn't show a building.

I think what you have here is probably better than CLIDATA, if I understand the dialogue correctly.  I hope that you could have many pictures for any station.  When you give the Station, then I hope the set of beginning dates could be provided, and each may correspond to a separate picture?  So you could have many pictures for any station and all those for a given station would be easily accessible.

The current picture didn't excite me, so I missed it completely.  It still doesn't look like a building.  I would like to see a picture also of a manual station (as in CLIDATA) so you can picture the surroundings of its location.

By the way, CLIDATA also has a section and table of Station visits.  I don't think that is yet in CLIMSOFT and is interesting.  

And they have Station Google maps:
![image](https://user-images.githubusercontent.com/11226469/44945173-4f057180-addb-11e8-957b-0ad2d0dc1605.png)

And Station Maps:
![image](https://user-images.githubusercontent.com/11226469/44945188-896f0e80-addb-11e8-98ee-387aeb9c2f3a.png)
